docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>If these haven't been introduced before it's a bit odd to ask what the deal is with something I haven't heard before. Maybe "What are value variables and why are they important?"</p><p> </p><p>And then I would add explanation as to what it is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
<p>Agree</p>

Reply:
@ricardoV94  Maybe you can help me with a few sentences for this intro! You have a much more in-depth knowledge than me to choose the right words :)